substrate/frame/support/procedural/src/construct_runtime/parse.rs

```diff
@@ -97,26 +97,29 @@ impl Parse for RuntimeDeclaration {
 		let pallets_token = pallets.token;
 
 		match convert_pallets(pallets.content.inner.into_iter().collect())? {
-			PalletsConversion::Implicit(pallets) =>
```

Comment:
All changes of this file look unrelated. I think you'll need to configure the rustfmt to comply with substrate

Reply:
didn't notice that it was relying on nightly settings, corrected